### PR TITLE
Moved reverse of the getPendingCreations to the AdminResolver instead…

### PR DESCRIPTION
**Pull request by Elweyn**
… to do it in the CreationConfirm page.

### Issues

- #1165 

### Todo

- [X] Remove 10000 of Frontend & Admin
- [X] Move array.reverse to backend
- [x] Use floats instead of integer 
- [x] Remove GraphQL error

### CODE REVIEW

**Review by ulfgebhardt — APPROVED** [review]

File: admin/src/graphql/createPendingCreation.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import gql from 'graphql-tag'
 export const createPendingCreation = gql`
   mutation (
     $email: String!
-    $amount: Int!
+    $amount: Float!
```
Comment (Mogge) [inline admin/src/graphql/createPendingCreation.js:6]: I do not like this at all. `amount` is definitely an integer. Did you try `parseInt(amount)` when setting the variables for the mutation?

File: admin/src/pages/CreationConfirm.vue [hunk 2]
```diff
@@ -70,6 +70,10 @@ export default {
           .catch((error) => {
             this.$toasted.error(error.message)
           })
+      } else {
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.vue:73]: I guess this is the case `confirmed`. Can we have a `switch` here that throws an error when no case matches?

**Review by Mogge — CHANGES_REQUESTED** [review]

File: admin/src/graphql/createPendingCreation.js (hunk 1 above)
Comment (Elweyn) [reply admin/src/graphql/createPendingCreation.js:6]: I though that it was something that we had to change!?
We want to create 1000,50 GDD or do we want to fix it to integers?

File: admin/src/pages/CreationConfirm.vue (hunk 2 above)
Comment (Elweyn) [reply admin/src/pages/CreationConfirm.vue:73]: Yeah this would be a good alternativ

File: admin/src/graphql/createPendingCreation.js (hunk 1 above)
Comment (Elweyn) [reply admin/src/graphql/createPendingCreation.js:6]: Was the point "Use float instead of integer" What did we mean with this?

**Review by Mogge — APPROVED** [review]

**Pull request merged**